.github/workflows/pr.yml

```diff
@@ -70,110 +70,112 @@ jobs:
-      - uses: coursier/setup-action@v1
-        if: ${{ env.shouldPerformBackendBuild == 'true' }}
-        with:
-          jvm: temurin:1.11.0.17
-      - uses: sbt/setup-sbt@v1
-      - name: Cache ivy packages
-        if: ${{ env.shouldPerformBackendBuild == 'true' }}
-        uses: actions/cache@v3
-        with:
-          path: |
-            ~/.ivy2/cache
-            ~/.cache/coursier
-            ~/.sbt
-          key: sbt-cache-${{ hashFiles('**/*.sbt') }}
-          restore-keys: sbt
-      - name: Build
-        if: ${{ env.shouldPerformBackendBuild == 'true' }}
-        shell: bash
-        run: CROSS_BUILD=true ./ciRunSbt.sh Compile/compile Test/compile
-      - name: Tar artifacts
-        if: ${{ env.shouldPerformBackendBuild == 'true' }}
-        shell: bash
-        run:  find . -wholename "**/target/**/*" -printf '%P\0' | tar --null -C '.' --files-from=- -czf 'target.tgz'
-      - name: Store target
-        if: ${{ env.shouldPerformBackendBuild == 'true' }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: build-target
-          path: target.tgz
-
-  build-fe:
-    name: BuildFrontend
-    runs-on: ubuntu-latest
-    needs: [ setup ]
-    env:
-      NUSSKNACKER_VERSION: ${{ needs.setup.outputs.nk_snapshot_version }}
-    steps:
-      - name: Cancel previous runs
-        if: github.event_name != 'push'
-        uses: styfle/cancel-workflow-action@0.11.0
-        with:
-          access_token: ${{ secrets.GITHUB_TOKEN }}
-      - uses: actions/checkout@v3
-      - name: Cache npm
-        id: cache-npm
-        uses: actions/cache@v3
-        with:
-          path: |
-            **/node_modules
-            ~/.cache/Cypress
-          key: ${{ hashFiles('designer/client/package-lock.json', 'designer/client/.nvmrc', 'designer/submodules/package-lock.json', 'designer/submodules/.nvmrc') }}
-      - name: Get Node.js version
-        id: nvm
-        run: echo "NODE_VERSION=$(cat designer/client/.nvmrc)" >> $GITHUB_OUTPUT
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
-      - name: Download node modules
-        if: steps.cache-npm.outputs.cache-hit != 'true'
-        run: |
-          npm ci --no-audit --prefix designer/client
-          npm ci --no-audit --prefix designer/submodules
-      - name: Build FE
-        run: (cd designer/client; npm run build)
-      - name: Build FE submodules
-        run: |
-          cd designer
-          cp -r client/.federated-types/nussknackerUi submodules/types/@remote
-          cd submodules
-          npm run build
-      - name: Tar fe artifacts
-        shell: bash
-        run:  tar -czf 'fe-dist.tgz' designer/client/dist designer/submodules/dist
-      - name: Store dist
-        uses: actions/upload-artifact@v4
-        with:
-          name: build-fe-dist
-          path: fe-dist.tgz
-
+#  build:
```

Comment:
what about this?

Reply:
I'll revert this changes It is for purpose of debugging some problem on CI